### ISSUE TITLE
Update to support GLPI 9.2

### ISSUE THREAD

**Issue by LuanPeixotop** [pluginsGLPI/room#27]
Please, can you update the plugin to support GLPI 9.2?

**Comment by C-Duv** [comment]
Support for 9.2 is planned, I'll have to look at what changes the core need for the plugin to work.

**Comment by institutocriap** [comment]
+1 wait to support GLPI 9.2....

**Comment by iamc** [comment]
+1

**Comment by iPaulis** [comment]
+1

**Comment by babasselegrand** [comment]
+1 thx

**Comment by iPaulis** [comment]
So, nothing yet? In my case, this plugin is the only thing preventing me to update to GLPI 9.2, but if there is no support, maybe I'll have to drop it and find an alternative.
I don't mind waiting, but I would like to know if there will ever be an update to support newer versions, or if the plugin is abandoned.

**Comment by Poiretc** [comment]
Hello,
Would this plugin still be relevant?
when do you think it will be updated?

Thanks.

**Comment by institutocriap** [comment]
+1

**Comment by C-Duv** [comment]
Hello all, thanks for the patience (I know that feeling when upgrading out GLPI instance is blocked by old plugin we have to use), I'll update the plugin this week-end :)

**Comment by C-Duv** [comment]
Please try the *evol/27-glpi_9.2_compatibility* branch, it should work on GLPI 9.2.

**Comment by institutocriap** [comment]
Where is donwload this version?

 

 

 

Atentamente,

Ivo Barbosa
Departamento de Informática

 <https://www.institutocriap.com/aniversario11/> 

 

 <https://www.facebook.com/InstitutoCriap>    <http://t.sidekickopen68.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XYg63RHDWW1pwWZR3LyBmCW653gwx56dHzGf2m4L2802?t=https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fcriap&si=6546342076481536&pi=eb55dad0-c95b-48ef-95df-a6c558d64ba5>    <http://t.sidekickopen68.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XYg63RHDWW1pwWZR3LyBmCW653gwx56dHzGf2m4L2802?t=https%3A%2F%2Fwww.youtube.com%2Fuser%2FInstitutoCRIAP&si=6546342076481536&pi=eb55dad0-c95b-48ef-95df-a6c558d64ba5>    <http://t.sidekickopen68.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XYg63RHDWW1pwWZR3LyBmCW653gwx56dHzGf2m4L2802?t=https%3A%2F%2Finstagram.com%2Finstituto_criap%2F&si=6546342076481536&pi=eb55dad0-c95b-48ef-95df-a6c558d64ba5> 

 

Aviso de Confidencialidade:

Esta mensagem de correio eletrónico e qualquer dos seus ficheiros anexos, caso existam, são confidenciais e destinados apenas à(s) pessoa(s) ou entidade(s) acima referida(s), podendo conter informação confidencial e/ou privilegiada, a qual não deverá ser divulgada, copiada, gravada ou distribuída nos termos da lei vigente. Se recebeu este e-mail por engano, agradecemos que nos contacte imediatamente, através de e-mail para  <mailto:geral@criap.com> geral@criap.com  e apague a comunicação original no seu sistema informático.

 

 

De: C-Duv [mailto:notifications@github.com] 
Enviada: 14 de janeiro de 2018 20:53
Para: pluginsGLPI/room <room@noreply.github.com>
Cc: institutocriap <informatica@criap.com>; Comment <comment@noreply.github.com>
Assunto: Re: [pluginsGLPI/room] Update to support GLPI 9.2 (#27)

 

Please try the evol/27-glpi_9.2_compatibility branch, it should work on GLPI 9.2.

**Comment by C-Duv** [comment]
From GitHub: https://github.com/pluginsGLPI/room/archive/evol/27-glpi_9.2_compatibility.zip

**Comment by C-Duv** [comment]
If no compatibility issue is reported here before 2018-01-23, I'll merge the branch into master and release a new version of the plugin.

**Comment by gustavoconde** [comment]
Hi, 
when doing test in glpi, the plugin menu is empty with bug. when I delete the folder, the glpi plugin normalizes.

**Comment by C-Duv** [comment]
I am running it on GLPI v9.2.2 without such issue.
Is there any error on the GLPI's logs?

**Comment by gustavoconde** [comment]
Hello,

I get put to work!
Now another question has arisen.


I created the room with your super admin user, doing test with user with profile "Self-Services", when I enter the option "Reservations", generates an error, am I doing something wrong?


![erro glpi](https://user-images.githubusercontent.com/37086407/37109537-0eb2cc3e-2219-11e8-99ca-a0f3f0fcfca4.png)

Thank you for your help

**Comment by zawaye** [comment]
I can install this plugin in GLPI 9.2.2 but I cannot restrict some user to see this room management menu. How can I do it to restrict some user seeing this menu, please?

**Comment by noanoe** [comment]
Hi!!! @gustavoconde How did you solve the error? 
I have the same error. When doing test in glpi, the plugin menu is empty with bug. when I delete the folder, the glpi plugin normalizes.

Thanks!!

**Comment by C-Duv** [comment]
@gustavoconde it seems the bug you encounter is coming from the *Generic Object* plugin.

@zawaye Please open new ticket and explain what you have at "Administration" > "Profiles" (/front/profile.form.php?id=xx), in the "Salles" tab.

**Comment by C-Duv** [comment]
I just released version 3.1.1 for GLPI v9.2 compatibility (which has no changes since _evol/27-glpi_9.2_compatibility_ branch).

Closing this issue.
